### DISCUSSION TITLE
Collaborator discussion

### DISCUSSION THREAD

**Discussion by BenjaminKey** [BenjaminKey/Tafe24S1T105_Ext#2] [General]
Hi Yan

Can you please provide your public key as per instructions in
SECTION C. (Individual work) 
- Individual member (i.e. the collaborator) needs to give the public key to the group leader to add into the key ring of the GitHub repository. The individual member must keep passphrase and the private key for future use.

Thanks
Benjamin

**Comment by BenjaminKey** [comment]
Hi Yan

I have completed SECTION B. (Team Work – Configure the VCS).
a.	Setting up the local repository
b.	Setting up the remote repository
c.	Prepare the development branch before major development

We are now ready to branch to our individual feature branches and continue on with SECTION C. (Individual work).
As we have discussed previously, I will work on the Mortgage Calculator & you will work on the Currency Convertor.

Github repository is ready for you to pull.

Thanks
Benjamin

**Comment by YanLi2024** [comment]
> Hi Yan
> 
> Can you please provide your public key as per instructions in SECTION C. (Individual work)
> 
> * Individual member (i.e. the collaborator) needs to give the public key to the group leader to add into the key ring of the GitHub repository. The individual member must keep passphrase and the private key for future use.
> 
> Thanks Benjamin

Hi Ben Thanks for the message. 
Here is my public key: 
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCSMl9ja3BYSlRDkr8+neX6+2ze/HxkaJRXE7sv/aUlWoo1J0KvlyjQ1UYPlgL0wqAJkDpbfWCntNXJBjSExsMB48fjUQjso+yuhrAHvw+09QEiFBtVA5HQakNzhOxvXEalWDa78wyGWi77L/uTXkDjy9QnOb9BO47v0O9w7muM82Bti2PW76lW0HMkd3OGkFoyazQA5cSM5+a0bWiXKodJD7k5UXxWa9YaL849zf4BtqfR2yQtOJwXtOx6JeYPeaOYbUwLmZTU9IHdi9deH09g/Xim2wfXbJHdqUIcY2csMsrtdfGArskSAVckK0t0mfIz7ShJdnYCG8KCaeJIwmYb rsa-key-20240310

**Comment by BenjaminKey** [comment]
Hi Yan

Public key added to the Github key ring

**Comment by BenjaminKey** [comment]
Hi Yan

FYI
When syncing or pushing use main as the remote branch - not master.

**Reply by YanLi2024** [reply]
Hi Ben
I cloned the files and I noticed that there was only main branch. I was considering asking you if we need to create a development branch as required in Section B. 
I started to write code for the currency conversion feature. almost done here. Now I see development branch. I will finish my part as soon as possible. Cheers.
